examples/patterns/headings/default.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: Headings / Default
+category: _patterns
+---
+
+<h1 class="p-heading--one">Heading one</h1>
+<h2 class="p-heading--two">Heading two</h2>
+<h3 class="p-heading--three">Heading three</h3>
+<h4 class="p-heading--four">Heading four</h4>
+<h5 class="p-heading--five">Heading five</h5>
+<h6 class="p-heading--six">Heading six</h6>
```

Comment:
At the minute, these styles reflect base styles but that might not always be the case. There's always a fallback for people to use without any classes but this section of the docs is specifically patterns - what you are describing is base styles.